src/update-feeds-loops/pending-transaction-info.ts

```diff
@@ -40,22 +40,29 @@ export const updatePendingTransactionsInfo = (
   } = getState();
 
   const currentTimestamp = Math.floor(Date.now() / 1000);
-  for (const feed of batch) {
-    const { dapiName, dataFeedId, decodedDapiName, updateParameters, dataFeedTimestamp } = feed;
-
-    const isFeedUpdatable = feedsToUpdate.some(
+  for (const { beaconsWithData, dapiName, dataFeedId, decodedDapiName, updateParameters, dataFeedTimestamp } of batch) {
+    const updatableFeed = feedsToUpdate.find(
```

Comment:
nit: can we preserve the old name?

Reply:
Sorry, I'm confused. You mean `const isFeedUpdatable`? I've changed it because it is no longer a boolean

Reply:
I see. Makes sense.